Tests/UQL/LexerTest.php

```diff
@@ -7,64 +7,70 @@
 class LexerTest extends \PHPUnit_Framework_TestCase
```

Comment:
I've changed previous test - one test method with 3 `foreach` loops testing 3 different cases, to 3 test methods with data provider.
And, of course, added "not in" operator test.